sycl/source/detail/scheduler/commands.cpp

```diff
@@ -64,6 +64,17 @@ bool CurrentCodeLocationValid() {
   return (FileName && FileName[0] != '\0') ||
          (FunctionName && FunctionName[0] != '\0');
 }
+
+void emitInstrumentation(uint32_t StreamID, QueueImplPtr Queue, uint64_t InstanceID, xpti_td* TraceEvent, uint16_t Type, const char *Txt) {
+  if (!(xptiCheckTraceEnabled(StreamID, Type) && TraceEvent))
+    return;
+  // Trace event notifier that emits a Type event
+  xpti::addMetadata(TraceEvent, "queue_id",
```

Comment:
move to the place where other metadata placed, adding it here trigger duplication path